apps/lasp/src/lasp_orset.erl

```diff
@@ -100,24 +100,55 @@ value(_, ORSet) ->
     value(ORSet).
 
 -spec update(orset_op(), actor(), orset()) -> {ok, orset()} |
+                                              {delta, orset()} |
                                               {error, {precondition, {not_present, member()}}}.
 update({add_by_token, Token, Elem}, _Actor, ORDict) ->
-    add_elem(Elem, Token, ORDict);
+    case application:get_env(lasp, delta_mode, true) of
```

Comment:
I would move this check to `lasp_type.erl`.  Everywhere an update is performed in Lasp, goes through `lasp_type:update`, so you can do this check once there, and have it call the delta one if the `delta_mode` is enabled.  

Make sense?